rskj-core/src/main/java/co/rsk/peg/utils/FederationUtils.java

```diff
@@ -0,0 +1,16 @@
+package co.rsk.peg.utils;
+
+import co.rsk.bitcoinj.script.Script;
+import co.rsk.peg.bitcoin.Standardness;
+
+public class FederationUtils {
+
+    public static boolean isScriptSizeValid(Script script) {
```

Comment:
It's better to have this method void and throw an exception (FederationCreationException could be) if the validation is not successful. That way we have less `if` statements in the code